pages/windows/rpcinfo.md

```diff
@@ -0,0 +1,20 @@
+# rpcinfo
+
+> List programs via RPC on remote computers.
+> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/rpcinfo>.
+
+- List all programs registered on the local computer:
+
+`rpcinfo`
+
+- List all programs registered on a remote computer:
+
+`rpcinfo /p {{computer_name}}`
+
+- Call a specific program on a remote computer using TCP:
```

Comment:
What's the default protocol ? We should mention that in parentheses.

Reply:
No default as far as I can tell from the Microsoft doc.

Reply:
Well, there has to be _some_ default protocol which is used. Strange that it's not mentioned in the docs.

Reply:
I can't find any information on this at all, so just installing it to test. 👍

Reply:
@agnivade the point is that from the doc it seems that you cannot connect to a remote computer if you don't specify the protocol. If you look at the doc, you can see `[/t <Node Program> [<version>]]`, there is no `<Node Program>` outside square brackets that are already including `/t` or `/u`.

Reply:
Oh I see ! Okay with me then